python/nav/web/templates/seeddb/_seeddb_netbox_form_content.html

```diff
@@ -2,54 +2,56 @@
 
 {% block form_fields %}
   <div class="row">
-    <div class="columns {{ columns_css_class|default:'large-12' }}">
-      <fieldset>
-        <legend>Inventory</legend>
-        {% include 'foundation-5/field.html' with field=form.ip %}
-        <div id="verify-address-feedback"></div>
-        {% include 'foundation-5/field.html' with field=form.room %}
-        {% include 'foundation-5/field.html' with field=form.category %}
-        {% include 'foundation-5/field.html' with field=form.organization %}
-      </fieldset>
-    </div>
+    {% with columns_css_class='large-4' %}
```

Comment:
Why are we setting `columns_css_class` here instead of hardcoding it in the divs directly?